modules/openapi-generator/src/test/java/org/openapitools/codegen/postman/PostmanCollectionCodegenTest.java

```diff
@@ -786,7 +786,7 @@ public void testQueryParameterDescription() throws IOException {
         output.deleteOnExit();
 
         final CodegenConfigurator configurator = new CodegenConfigurator()
-                .setGeneratorName("postman-v2")
+                .setGeneratorName("postman-collection")
                 .setInputSpec("./src/test/resources/SampleProject.yaml")
```

Comment:
@gcatanese can you please take a look at this line when you've time? SampleProject.yaml is a config file but not an OpenAPI spec.

Reply:
Same issue in line 766

Reply:
Sorry, I made a mistake re-using a test on a local implementation (I did miss the CI failures though). The test file is `"src/test/resources/3_0/postman-collection/SampleProject.yaml"`

Reply:
can you please PM me via Slack to further discuss this?